tests/utils_tests/test_html.py

```diff
@@ -66,6 +69,37 @@ def test_format_html(self):
             "&lt; Dangerous &gt; <b>safe</b> &lt; dangerous again <i>safe again</i>",
         )
 
+    def test_format_html_join_with_positional_arguments(self):
+        self.assertEqual(
+            format_html_join(
+                "\n", "<li>Id: {}</li><li>Text: {}</li>", ((1, "Text1"), (2, "Text2"))
```

Comment:
One argument per line for clarity. Also, use a list-of-tuples, per [this rule of thumb](https://stackoverflow.com/a/626871/1427135).

```suggestion
                "\n",
                "<li>Id: {}</li><li>Text: {}</li>",
                [(1, "Text1"), (2, "Text2")],
```